test/package/package_b/__init__.py

```diff
@@ -0,0 +1,15 @@
+__import__("subpackage_1", globals(), fromlist=["PackageBSubpackage1Object_0"], level=1)
+__import__("package_a", fromlist=[])
```

Comment:
when doing automatic test case discovery, `unittest` will execute all `__init__.py` files to try to find more tests to run. This import will fail because `package_a` is not in the module search path at that point.

I suggest changing this to importing a subpackage, which will probably just work (you can run `python test/test_package.py` to check).